spec/acceptance/class_plugin_python_spec.rb

```diff
@@ -57,7 +57,7 @@
       # Run it twice and test for idempotency
       apply_manifest(pp, catch_failures: true)
       apply_manifest(pp, catch_changes: true)
-      shell('sleep 10')
+      shell('sleep 30')
```

Comment:
why did you need to increase this? Does collectd now start slower?

Reply:
To try to get the tests to pass. Wondered if it was some kind of race
condition